flatfs.go

```diff
@@ -601,6 +638,24 @@ func (fs *Datastore) putMany(data map[datastore.Key][]byte) error {
 }
 
 func (fs *Datastore) Get(key datastore.Key) (value []byte, err error) {
+	value, err = fs.get(key)
```

Comment:
Hm. Maybe push the retry loop all the way down into `readFile`? That could save us some work.